feature-libs/organization/administration/core/store/actions/b2b-user.action.ts

```diff
@@ -196,12 +223,20 @@ export class LoadB2BUsers extends StateUtils.EntityLoadAction {
 }
 
 export class LoadB2BUsersFail extends StateUtils.EntityFailAction {
-  error: ErrorActionType = this.payload.error;
```

Comment:
Note: such lines are not needed in classes that extend from `StateUtils.EntityFailAction`